client/dashboard/sites/settings-subscription-gifting/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { WPCOM_FEATURES_SUBSCRIPTION_GIFTING } from '@automattic/calypso-products';
```

Comment:
I would avoid importing this package just for a constant, this package looks like a kitchen sink package and these are not great packages in general. It's not really clear what dependencies we're breaking by doing this. calypso-i18n is one that we want to avoid for instance.

Reply:
The question becomes, why is it a package in this case? I'm trying to add more clarity/guidelines about what makes a good package [here](https://github.com/Automattic/wp-calypso/pull/103207). Unfortunately most existing packages don't mint the criteria for me.

 - Like for me, I'd rather have a package with just the features constants
 - Or just have a file in `client/dashboard`. I don't really understand what the current package is bringing?

Reply:
Also for me, this is part of the "data" layer, so I can see a file in `client/dashboard/data/constants` or something like that.

Reply:
I agree, I would avoid the import too 🙂

Reply:
Okay, for now I simply duplicate the constant in the `dashboard` codebase: https://github.com/Automattic/wp-calypso/pull/103227/commits/c4033cf6697bd71875c9d9c976095797556b5e21